packages/reports/addon/components/filter-values/date.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ *   {{filter-values/date
+ *       filter=filter
+ *       request=request
+ *       onUpdateFilter=(action 'update')
+ *   }}
+ */
+import Component from '@ember/component';
+import Moment from 'moment';
+import layout from '../../templates/components/filter-values/date';
+import { computed, get } from '@ember/object';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {Array} classNames
+   */
+  classNames: ['date'],
```

Comment:
I feel like we need a better name than date

Reply:
`date-range` is the existing one so I figured `date` was a logical choice for a single date.